content/docs/reference/tutorials/aws/tutorial-ec2-webserver.md

```diff
@@ -190,9 +325,9 @@ Before moving on, let's tear down the resources that are part of our stack.
 
 ## Summary
 
-In this tutorial, we saw how to use Pulumi programs to create and manage cloud resources in AWS, using regular JavaScript and NPM packages. To preview and update infrastructure, use `pulumi up`. To clean up resources, run `pulumi destroy`.
+In this tutorial, we saw how to use Pulumi programs to create and manage cloud resources in AWS, using our programming language of choice and its corresponding dependency manager. To preview and update infrastructure, use `pulumi up`. To clean up resources, run `pulumi destroy`.
```

Comment:
Nit: I think "package manager" is the more widely used term than "dependency manager".

```suggestion
In this tutorial, we saw how to use Pulumi programs to create and manage cloud resources in AWS, using our programming language of choice and its corresponding package manager. To preview and update infrastructure, use `pulumi up`. To clean up resources, run `pulumi destroy`.
```